elasticdl/python/master/task_dispatcher.py

```diff
@@ -10,12 +10,15 @@
 class _Task(object):
     """Internal representation of a task"""
 
-    def __init__(self, *, shard_name, start, end, type, model_version=-1):
+    def __init__(
+        self, shard_name, start, end, type, model_version=-1, **kwargs
+    ):
         self.shard_name = shard_name
         self.start = start
         self.end = end
         self.type = type
         self.model_version = model_version
+        self.extended_config = kwargs
```

Comment:
Maybe `"extra_config"` is better?

Reply:
I think 'extra_config' and 'extended_config' are both fine. I prefer to keep the current name. :)